rollout/controller_test.go

```diff
@@ -1055,15 +1060,15 @@ func TestPodTemplateHashEquivalence(t *testing.T) {
 	var err error
 	// NOTE: This test will fail on every k8s library upgrade.
 	// To fix it, update expectedReplicaSetName to match the new hash.
-	expectedReplicaSetName := "guestbook-7b5c4d46d9"
+	expectedReplicaSetName := "guestbook-7df8bcd895"
 
 	r1 := newBlueGreenRollout("guestbook", 1, nil, "active", "")
 	r1Resources := `
 limits:
-  cpu: 150m
+  cpu: 2000m
   memory: 8192M
 requests:
-  cpu: 2000m
+  cpu: 150m
```

Comment:
Why do you change the limits and requests here and below?

Reply:
The CPU request must be less than the limit in order to pass the Rollout Spec Validation.